pkg/oc/admin/diagnostics/diagnostics/util/util.go

```diff
@@ -1,9 +1,42 @@
 package util
 
+import (
+	"fmt"
+	"strings"
+)
+
 var (
 	AdminKubeConfigPaths = []string{
 		"/etc/openshift/master/admin.kubeconfig",           // enterprise
 		"/openshift.local.config/master/admin.kubeconfig",  // origin systemd
 		"./openshift.local.config/master/admin.kubeconfig", // origin binary
 	}
 )
+
+func TrimRegistryPath(image string) string {
```

Comment:
I approved the same change in NetworkCheck but I'm having second thoughts. I'm not sure trimming the registry is the right approach. It does not seem like it should be necessary. I'm going to tinker a little bit to make sure I understand what's really happening.

Reply:
I am fine with trimming the registry from the client's default, which would leave it open to the node to prepend a registry. But I want to make sure the templating gets represented in the help and the user's template expanded the same, and it looks like that doesn't happen here.

Reply:
Actually, see https://bugzilla.redhat.com/show_bug.cgi?id=1475680#c7